courses/machine_learning/deepdive2/image_classification/labs/2_mnist_models.ipynb

```diff
@@ -17,9 +17,18 @@
     "First things first. Configure the parameters below to match your own Google Cloud project details."
```

Comment:
<p>In title, replace "Cloud ML Engine" with "Cloud AI Platform".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/997/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Hi @gstripling ,

I have updated the changes mentioned above. The PR can be merged now.

Feel free to let me know if anything else needs to be updated.

-Thanks